src/views/report/ScriptReportsOverview/index.tsx

```diff
@@ -162,6 +163,8 @@ const defaultSortedColumn: ISortedColumn<IColumnNames> = {
     sortType: SortType.String,
 };
 
+let pageInterval: ReturnType<typeof setInterval>;
```

Comment:
pageInterval is a type of number and not `ReturnType<typeof setInverval`.
pageInterval should be a `const`